apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/entity/condition/ConditionCode.java

```diff
@@ -107,13 +113,33 @@ public class ConditionCode implements Serializable {
     private Character morbReportEnableInd;
 
     @Column(name = "port_req_ind_cd")
-    private Character portReqIndCd;
+    private String portReqIndCd;
```

Comment:
The database defines this as a char(1). 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/b710d695-ae87-4544-9d64-1d58c6ef7794)

we should change the input to a Character instead of changing the database entity to hold a String